_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -389,6 +389,9 @@ By default it is deployed using [systemd socket activation](http://0pointer.de/b
 
 ### Deprecations
 
+* Puppetrun feature is being removed from Foreman core since version 2.2 ([more info](https://community.theforeman.org/t/dropping-run-puppet-button-with-a-replacement-using-remote-execution-plugin/12781/25)). However, the function of Puppetrun can be subsided by Remote Execution plugin. There is two options:
+    * There is `Puppet run once - SSH default` job template that takes `puppet_options` as parameter before it's invoked
```

Comment:
I believe there are also ansible and salt templates for the same action in those plugins

Reply:
Yes, you're right. But I think it's out of this PR because we're discussing here stuff related to puppet. What do you think @ares?

Reply:
how about:
```suggestion
    * There is a `Puppet run once - SSH default` job template that takes `puppet_options` as parameter before it's invoked. There are also similar templates for other Remote Execution providers.
```